packages/react-native-bpk-component-calendar/stories.js

```diff
@@ -0,0 +1,198 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import React, { Component } from 'react';
+import { View } from 'react-native';
+import { storiesOf } from '@storybook/react-native';
+import BpkTextInput from 'react-native-bpk-component-text-input';
+import BpkPicker, { BpkPickerItem } from 'react-native-bpk-component-picker';
+import BpkSelect from 'react-native-bpk-component-select';
+
+import CenterDecorator from '../../storybook/CenterDecorator';
+import BpkCalendar, {
+  SELECTION_TYPES,
+  type BpkCalendarSelectionType,
+} from './index';
+
+/* eslint-disable react/no-multi-comp */
+
+class BpkCalendarExample extends Component<
+  {
+    onChangeSelectedDates?: (Date[]) => mixed,
+    selectionType: BpkCalendarSelectionType,
+  },
+  {
+    selectedDates: Date[],
+  },
+> {
+  constructor(props) {
+    super(props);
+    this.state = { selectedDates: [] };
+  }
+
+  render() {
+    return (
+      <BpkCalendar
+        locale="en_GB"
+        selectionType={this.props.selectionType}
+        onChangeSelectedDates={newDates => {
+          if (this.props.onChangeSelectedDates) {
+            this.props.onChangeSelectedDates(newDates);
+          }
+          this.setState({ selectedDates: newDates });
+        }}
+        style={{ flexGrow: 1 }}
+        selectedDates={this.state.selectedDates}
+      />
+    );
+  }
+}
+
+class ExampleWithLinkedInputs extends Component<
+  {
+    range: boolean,
+  },
+  {
+    selectedDates: Date[],
+  },
+> {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      selectedDates: [],
+    };
+  }
+
+  onChangeText = value => {
+    let { selectedDates } = this.state;
+    if (value === '') {
+      selectedDates = [];
+    }
+    this.setState({ selectedDates });
+  };
+
+  handleNewDates = newDates => {
+    this.setState({
+      selectedDates: newDates,
+    });
+  };
+
+  render() {
+    const { range } = this.props;
+    const { selectedDates } = this.state;
+    return (
+      <View>
```

Comment:
```suggestion
      <View style={{ flex: 1 }} >
```

Reply:
Is this needed?

Reply:
![screenshot 2019-01-23 at 14 37 27](https://user-images.githubusercontent.com/30267516/51613896-6d037800-1f1c-11e9-95f6-40f48a0eab5a.png)

Reply:
Hmm I thought I had resolved that. Good catch

Reply:
Btw, I've improved (or so I think) this a bit in my branch so I'll add that as soon as this gets merged